docs/configuration.md

```diff
@@ -86,6 +86,31 @@ changed here.
 
     Default is `prefer`.
 
+`PAPERLESS_DBSSLROOTCERT=<ca-path>`
```

Comment:
Each of these should note the default is unset (for example as `PAPERLESS_DB_TIMEOUT` does), which disables usage of certs.